src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -95,7 +125,13 @@ public void stopMotors() {
 		frontRightMotor.set(0);
 		backRightMotor.set(0);
 	}
+	public double getHeading() {
+		return -Math.IEEEremainder(gyro.getAngle(), 360);
+	}
 
+	public double getAverageEncoderDistance() {
```

Comment:
Do we ever reset the encoder counts? If not they will continue counting up through turns/subsequent operations. Also we should probably reset them on init.